gatt_blacklist_policy.md

```diff
@@ -0,0 +1,82 @@
+# Policy for adding UUIDs to the GATT Blacklist
+
+## UUIDs are added through Github issues in this repository
+
+There is currently no minimum or maximum time between
+when the issue is filed and when the new UUID is added to the blacklist.
```

Comment:
How's this? I don't really want to mention Chrome specifically, but all implementations should have some way to push these changes quickly.